vtex/toolbelt
# Add DEBUG_CP env var for debugging child_processes

#### What is the purpose of this pull request?
Just as the title says

#### How should this be manually tested?
`yarn watch` this version and run commands like `DEBUG_CP=* vtex-test workspace reset`
(I added `export DEBUG_CP=*` on my `.bashrc` to always see child processes output)

#### Types of changes

#### Chores checklist
